src/analysis/schwartz_space.lean

```diff
@@ -535,7 +556,20 @@ mk_clm (fderiv ℝ)
 @[simp] lemma fderiv_clm_apply (f : 𝓢(E, F)) (x : E) : fderiv_clm 𝕜 f x = fderiv ℝ f x :=
 rfl
 
-end fderiv
+/-- The 1-dimensional derivative on Schwartz space as a continuous `𝕜`-linear map. -/
+def deriv_clm : 𝓢(ℝ, F) →L[𝕜] 𝓢(ℝ, F) :=
```

Comment:
Possibly worth adding to the module docstring

Reply:
done